grafana-charts/cstorvolumereplicasdashboard.json

```diff
@@ -0,0 +1,1383 @@
+{
+	"dashboard": {
+		"__inputs": [
+			{
+				"name": "DS_OPENEBS_PROMETHEUS",
+				"label": "openebs Prometheus",
+				"description": "",
+				"type": "datasource",
+				"pluginId": "prometheus",
+				"pluginName": "Prometheus"
+			}
+		],
+		"__requires": [
+			{
+				"type": "grafana",
+				"id": "grafana",
+				"name": "Grafana",
+				"version": "5.1.3"
+			},
+			{
+				"type": "panel",
+				"id": "graph",
+				"name": "Graph",
+				"version": ""
+			},
+			{
+				"type": "datasource",
+				"id": "prometheus",
+				"name": "Prometheus",
+				"version": "1.0.0"
+			},
+			{
+				"type": "panel",
+				"id": "singlestat",
+				"name": "Singlestat",
+				"version": ""
+			},
+			{
+				"type": "panel",
+				"id": "text",
+				"name": "Text",
+				"version": ""
+			}
+		],
+		"annotations": {
+			"list": [
+			]
+		},
+		"editable": true,
+		"gnetId": null,
+		"graphTooltip": 0,
+		"id": null,
+		"iteration": 1542963642391,
+		"links": [
+			{
+				"icon": "doc",
+				"includeVars": true,
+				"keepTime": true,
+				"tags": [
+				],
+				"targetBlank": true,
+				"title": "report",
+				"tooltip": "",
+				"type": "link",
+				"url": "${MAYA_URL}/reporter/?dashboard=644d0435-03e1-4da3-b768-7968f526cf68"
```

Comment:
@Ab-hishek What should we replace here. ?

@survivant I guess this is the part that you mentioned?

Reply:
Yes Akhil. Looking into this.

Reply:
yes.